default.env

```diff
@@ -84,3 +79,22 @@ RCON_PORT=25575
 REGION=
 USEAUTH=true
 BAN_LIST_URL=https://api.palworldgame.com/api/banlist.txt
+
+### Webhook Settings
```

Comment:
Move to after setttings-mode

#Container-Settings
x
y
z
#Gameserver-Settings
a
b
c